utils/documentOps.py

```diff
@@ -157,18 +154,24 @@ def parse_file(file_name):
         if p is not None:
             assert op is not None, "Min without op."
             assert op not in min_dict, "Redefinition of min for op " + op
-            min_dict[op] = int(p[1])
+            op_min = max(int(p[1]), min_opset_default)
```

Comment:
Note: this approach requires MIN to come before UNSUPPORTED, which is not necessary but ok.